app/models/learning_progress.py

```
@@ -0,0 +1,46 @@
+import enum
+from datetime import date
+from typing import TYPE_CHECKING, List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)